src/assets/styles/fonts.scss

```diff
@@ -83,3 +83,101 @@ $lob-assets-url: 'https://s3-us-west-2.amazonaws.com/public.lob.com';
   font-weight: 800;
   src: url('#{$lob-assets-url}/fonts/larsseit/larsseit-extrabold-italic.otf') format('opentype');
 }
+
+/* Messina */
+
+@font-face {
+  font-family: Messina;
+  font-style: normal;
+  font-weight: 300;
+  src: url('#{$lob-assets-url}/fonts/messina/MessinaSansWeb-Light.woff2') format('woff2'),
+    url('#{$lob-assets-url}/fonts/messina/MessinaSansWeb-Light.woff') format('woff');
+}
```

Comment:
importing both woff and woff2 for browser compatibility
[WOFF - Web Open Font Format shows a browser compatibility score of 100. ](https://www.lambdatest.com/web-technologies/woff)
[WOFF 2.0 - Web Open Font Format shows a browser compatibility score of 88.](https://www.lambdatest.com/web-technologies/woff2)